### PR TITLE
FORMS731: Draft Url access bug

### PR DESCRIPTION
**Pull request by bcvesalink**
# Description
Fix when user can access still the Draft url even if submission has been submitted already

## Types of changes

Bug fix (non-breaking change which fixes an issue) 

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have run the npm script lint on the frontend and backend
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have approval from the product owner for the contribution in this pull request

### CODE REVIEW

**Review by jasonchung1871 — APPROVED** [review]

**Comment by jasonchung1871** [comment]
Oh, it failed to complete the test!

File: app/frontend/src/components/forms/submission/UserSubmission.vue [hunk 1]
```diff
@@ -26,16 +27,41 @@ export default {
...
+      this.$router.push({
+        name: 'UserFormView',
+        query: {
+          s: this.formSubmission?.id,
+        },
+      });
+      this.addNotification({
+        ...NotificationTypes.WARNING,
+        message:
+          'Requsted submission is already submitted, redirecting to "View" page',
```
Comment (timisenco2015) [inline app/frontend/src/components/forms/submission/UserSubmission.vue:62]: This needs to be translated

Comment (bcvesalink) [reply app/frontend/src/components/forms/submission/UserSubmission.vue]: @timisenco2015  added

**Comment by bcvesalink** [comment]
> Oh, it failed to complete the test!

@jasonchung1871 tests fixed

**Pull request merged**